examples/cloud/gettingStartedWeb.php

```diff
@@ -75,87 +75,19 @@
  *
  * ## Class
  */
-require_once(__DIR__ . "/exampleUtils.php");
-require_once(__DIR__ . "/../../vendor/autoload.php");
-
-use fiftyone\pipeline\devicedetection\DeviceDetectionPipelineBuilder;
-use fiftyone\pipeline\core\Logger;
-use fiftyone\pipeline\core\Utils;
 
+require_once(__DIR__ . '/../../vendor/autoload.php');
```

Comment:
do we need to use this require here?

Reply:
Yes, otherwise running the examples according to the instructions in the readme will result in a `Class ... not found` error.